uber/templates/registration/form.html

```diff
@@ -271,6 +271,8 @@
                 {{ macros.checkbox(attendee, 'got_merch', label='Yes') }}
             </div>
         </div>
+    {% else %}
+        <input type="hidden" name="got_merch" value="{{ '1' if attendee.got_merch else '0' }}" />
```

Comment:
`{{ attendee.got_merch }}` is a boolean, so it would output `value="True"` or `value="False"`. If you look at the code for `def apply()`, you'll see it's expecting boolean checkboxes to have a value of `1` or `0`. It attempts to do `int(float(value))` which invariably chokes on `True` or `False`.